proto/v1beta1/grafeas.proto

```diff
@@ -171,11 +188,17 @@ service GrafeasV1Beta1 {
     option (google.api.http) = {
       get: "/v1beta1/{parent=projects/*}/occurrences:vulnerabilitySummary"
     };
+    option (google.api.method_signature) = "parent,filter";
   };
 };
 
 // An instance of an analysis type that has been found on a resource.
 message Occurrence {
+  option (google.api.resource) = {
+    type: "grafeas.grafeas.io/Occurrence"
```

Comment:
I guessed on this service name here. I'd appreciate some guidance from the team. More info can be found [here](https://aip.dev/123#terminology).

Reply:
This is fine.

Reply:
Based on the conventions adopted by Google and Kubernetes jointly, defined [here](https://github.com/kubernetes/community/blob/c1745f728f9eda89d506134fba96adebb35307b3/contributors/devel/sig-architecture/api-conventions.md), I changed it to the top-level domain, removing the stuttering subdomain.